test/prompt.select.js

```diff
@@ -149,23 +149,19 @@ describe('select', function() {
       });
 
       prompt.once('run', async() => {
-        try {
-          let { state, symbols } = prompt;
-          let pointer = cyan(symbols.pointer);
-          let hint = dim('(this is a hint)');
-          let expected = `${pointer} ${cyan.underline('A')}\n  BB ${hint}\n  CCC\n  DDDD`;
-          let actual = await prompt.renderChoices();
-          assert.equal(actual, expected);
-          await prompt.submit();
-        } catch (err) {
-          cb(err);
-        }
+        let { symbols } = prompt;
+        let pointer = cyan(symbols.pointer);
+        let hint = dim('(this is a hint)');
+        let expected = `${pointer} ${cyan.underline('A')}\n  BB ${hint}\n  CCC\n  DDDD`;
+        let actual = await prompt.renderChoices();
+        assert.equal(actual, expected);
+        await prompt.submit();
       });
 
-      prompt.run().then(() => cb()).catch(cb);
+      prompt.run();
     });
 
-    it.skip('should render a list of choices with the correct styles', cb => {
+    it('should render a list of choices with the correct styles', () => {
```

Comment:
Un-skipped a few tests like this one. 
They were running fine. 
Tell me if I should comment them again.